plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -288,6 +298,45 @@ static void ffmpeg_source_open(struct ffmpeg_source *s)
 	}
 }
 
+static void ffmpeg_source_start(struct ffmpeg_source *s)
+{
+	if (!s->media_valid)
+		ffmpeg_source_open(s);
+
+	if (s->media_valid) {
+		mp_media_play(&s->media, s->is_looping);
+		if (s->is_local_file)
+			obs_source_show_preloaded_video(s->source);
+		set_media_state(s, OBS_MEDIA_STATE_PLAYING);
+		obs_source_media_started(s->source);
+	}
+}
+
+static void *ffpmeg_source_reconnect(void *data)
```

Comment:
Use `ffmpeg_source_reconnect` instead of `ffpmeg_source_reconnect` here and throughout the PR.

Reply:
Fix typo:

```suggestion
static void * ffmpeg_source_reconnect(void *data)
```